### ISSUE TITLE
AI-Powered Multi-Protocol Message Generator

### ISSUE THREAD

**Issue by amiable-dev** [amiable-dev/conductor#3]
# AI-Powered Message Generator

Create a message generator tool that produces realistic MIDI and Gamepad input sequences using AI assistance. Supports AI-generated patterns (via MCP), manual message creation, and record-replay for capturing real device input.

## Dependencies

**Blocked by**: #2 (MCP Server for Protocol Specs)

### 1. AI Generation Mode
Natural language prompts → realistic test sequences

```bash
conductor-generator --ai "Generate fighting game combo: quarter-circle forward + punch"
conductor-generator --ai "Simulate 5 seconds of FPS gameplay with WASD + mouse"
conductor-generator --ai "Create drum pattern with velocity dynamics"
```

### 2. Manual Mode
CLI flags for precise control

```bash
conductor-generator --gamepad --button 128 --duration 100ms
conductor-generator --midi --note 60 --velocity 100
```

### 3. Record-Replay
Capture real device input

```bash
conductor-generator --record --device "Xbox Controller" --output test.json
conductor-generator --replay test.json
```

## Implementation

- Virtual MIDI/gamepad device creation
- AI integration via MCP (#2)
- JSON sequence format
- Direct IPC to daemon (bypass virtual device on macOS)
- Pre-generated test library

## Labels

`enhancement`, `testing`, `ai`, `v3.1`

**Comment by amiable-dev** [comment]
## Strategic Pivot: Enhanced and Incorporated into #5

This issue's core concept has been **enhanced and incorporated** into the crowdsourced pattern platform (#5).

**Superseded by**: #5 (Crowdsourced Pattern Platform)

### Evolution

**Original scope** (this issue):
- AI-powered message generator
- Virtual device simulation
- Manual/AI/Record modes

**Enhanced scope** (#5):
- All features from this issue **PLUS**:
- Crowdsourced real pattern library
- Cloud-based pattern marketplace
- AI trains on actual human behavior (not synthetic)
- Community-driven test data

### What's Being Built

The message generator concept evolves into:

1. **conductor-capture** (Phase 1): Record real input with consent
2. **conductor-generator** (Phase 4): Enhanced with cloud pattern search
3. **conductor-cloud** (Phase 2): Pattern marketplace backend
4. **conductor-cloud-web** (Phase 5): Web UI for browsing patterns

### Why This Is Better

- Real human patterns > synthetic AI patterns
- Network effects (more users = better data)
- Community knowledge sharing
- Training data for AI fine-tuning

### Status

This issue is superseded but its core features are enhanced in #5.
